tests/ha/hawk_gui.pm

```diff
@@ -83,7 +79,10 @@ sub run {
     assert_script_run "mkdir -m 1777 $path";
     assert_script_run "xhost +";
     barrier_wait("HAWK_GUI_CPU_TEST_START_$cluster_name");
-    my $docker_cmd = "docker run --rm --name test --ipc=host -v /tmp/.X11-unix:/tmp/.X11-unix -e DISPLAY=\$DISPLAY -v \$PWD/$path:/$path ";
+    assert_script_run("ls");
+    become_root;
```

Comment:
why is needed? Is it possible to add an explanation as code comment?

Reply:
It is needed because podman doesn't have the necessary permissions, so I'm running as root to make it work as intended. I'll add a comment